liveIT-db/schema/restaurants/Cuisine.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE Cuisine (
+id SMALLINT AUTO_INCREMENT,
+type VARCHAR(50) NOT NULL,
+PRIMARY KEY(id)
+) ENGINE = INNODB;
```

Comment:
what is this used for?

Reply:
its precautionary! Storage engine used for mySQL. I can remove it...